src/runtime_src/core/common/drv/kds_core.c

```diff
@@ -1076,13 +1081,53 @@ _kds_fini_client(struct kds_sched *kds, struct kds_client *client,
 	mutex_unlock(&client->lock);
 }
 
+
+static inline void
```

Comment:
Does this function really need to be "inlined"?

Reply:
Removed. Thanks.